package.json

```diff
@@ -67,6 +67,7 @@
     "@typescript-eslint/parser": "^5.46.1",
     "autoprefixer": "^10.4.13",
     "babel-plugin-named-exports-order": "^0.0.2",
+    "eslint-config-react-app": "^7.0.1",
```

Comment:
why the need to install this?...according to[ the docs ](https://www.npmjs.com/package/eslint-config-react-app)it should have already been incldued in CRA...

Reply:
Installed this because when I would run npm start I would get a "failed to load config react-app to extend form" error and this was the solution that I found on Google. https://www.google.com/search?q=failed+to+load+config+react-app+to+extend+from&oq=failed+to+load+config&aqs=chrome.1.69i57j0i20i263i512j0i512j0i20i263i512j0i512l6.3729j0j7&sourceid=chrome&ie=UTF-8